mesa/space.py

```diff
@@ -909,7 +907,7 @@ def remove_agent(self, agent: Agent) -> None:
     def get_neighbors(
         self, pos: FloatCoordinate, radius: float, include_center: bool = True
     ) -> list[Agent]:
-        """Get all objects within a certain radius.
+        """Get all agents within a certain radius.
```

Comment:
I suppose this is true in general -- a space contains only agents. It's always agents, even for fixtures.

Reply:
Even if there are such models, the current space implementation doesn't accommodate them yet (also evident from the Mypy typing). As such, I'm merging now.